assets/js/components/promoBanner/promo-banner.js

```diff
@@ -0,0 +1,109 @@
+const chooseCookieByMonth = () => {
```

Comment:
Sounds good! So just to clarify, would calling a new JSON file with keys referring to the months and values referring to dictionary values of the cookie data of that month to retrieve the cookie of the month work in place of having all of the cases?

Reply:
I think it might be good to stay consistent with the way that `components/cookieCards/cookies.json` is defined and handled as a point of reference.

My first instinct was also to build out a JSON that looks like `{"january": { "name": "smores", ...` but looking at the patterns that the other component was built like, I think consistency is more important here.

That would look something like:
```js
[
    {
        month: 'January',
        name: "S'mores",
        image: "smores.jpg",
        description: "Feeling adventurous? Try this month's signature cookie..."
    },
    {
        month: 'February',
        name: "Adventurefuls",
        image: "adventurefuls.jpg",
        description: "Feeling adventurous? Try this month's signature cookie..."
    },
];
```